### PR TITLE
Add a Home button

### PR DESCRIPTION
**Pull request by sebastiantoh**
Looks pretty ugly: 
- Animation of new view comes in from the right
- Toolbar doesn't align between the two views (editing view and canvas selection view)

### CODE REVIEW

File: Dynavity/Dynavity/view/ToolbarView.swift [hunk 1]
```diff
@@ -190,15 +190,31 @@ struct ToolbarView: View {
         .background(viewModel.canvasMode == .lasso ? Color.UI.grey : nil)
     }
 
+    private var backButton: some View {
+
```
Comment (ianyong) [inline Dynavity/Dynavity/view/ToolbarView.swift]:
```suggestion
```

File: Dynavity/Dynavity/view/ToolbarView.swift [hunk 2]
```diff
@@ -190,15 +190,31 @@ struct ToolbarView: View {
...
+        // TODO: replace canvases parameter with actual canvases
+        NavigationLink(destination: CanvasSelectionView(canvases: [])
+                        .navigationBarHidden(true)
+                        .navigationBarBackButtonHidden(true)) {
+            Image(systemName: "house")
+        }
+        .navigationBarHidden(true)
+
```
Comment (ianyong) [inline Dynavity/Dynavity/view/ToolbarView.swift]:
```suggestion
```

**Review by ianyong — DISMISSED** [review]
LGTM! Would it be possible to have the canvas animation slide in and out instead of perpetually sliding in though? (This isn't important so feel free to merge this PR in anyway.)

**Comment by sebastiantoh** [comment]
> LGTM! Would it be possible to have the canvas animation slide in and out instead of perpetually sliding in though? (This isn't important so feel free to merge this PR in anyway.)

Unfortunately, I don't think SwiftUI supports modiffying the transition / animation :(

**Pull request merged**